packages/jansson.rb

```diff
@@ -27,6 +19,6 @@ def self.build
   end
```

Comment:
Shouldn't this also abstract the library directory?  I guess my question is why comment the other options?

Reply:
They were like that and I'm not familiar with the package so I left it how it was.

Reply:
Yeah, a lot of these older packages were built before we knew about the abstractions.  I wonder if we need to create a build directory?
```
Dir.mkdir 'build'
Dir.chdir 'build' do
  system 'cmake',
              "-DCMAKE_INSTALL_PREFIX=#{CREW_PREFIX}"
              "-DCMAKE_INSTALL_LIBDIR=#{ARCH_LIB}",
              '..'
end
```